inception-imls-opennlp/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/ner/OpenNlpNerRecommender.java

```diff
@@ -157,10 +160,16 @@ public double evaluate(List<CAS> aCasses, DataSplitter aDataSplitter)
                 break;
             }            
         }
+        
+        int testSetSize = testSet.size();
+        int trainingSetSize = trainingSet.size();
+        result.setTestSetSize(testSetSize);
+        result.setTrainingSetSize(trainingSetSize);
 
-        if (trainingSet.size() < 2 || testSet.size() < 2) {
+        if (trainingSetSize < 2 || testSetSize < 2) {
             LOG.info("Not enough data to evaluate, skipping!");
-            return 0.0;
+            result.setDefaultScore(0.0);
```

Comment:
Setting the score to 0.0 seems redundant here because the value is already initialized in the `EvaluationResult` with 0.0. Might be a good idea to include a flag in the evaluation result whether the evaluation was actually performed or skipped.